generators/server/templates/pom.xml.ejs

```diff
@@ -756,38 +756,13 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-resources-plugin</artifactId>
                     <version>${maven-resources-plugin.version}</version>
-                    <executions>
-                        <execution>
-                            <id>default-resources</id>
-                            <phase>validate</phase>
-                            <goals>
-                                <goal>copy-resources</goal>
-                            </goals>
-                            <configuration>
-                                <outputDirectory>${project.build.directory}/classes</outputDirectory>
-                                <useDefaultDelimiters>false</useDefaultDelimiters>
-                                <delimiters>
-                                    <delimiter>#</delimiter>
-                                </delimiters>
-                                <resources>
```

Comment:
Maybe you will want to keep a resources definition in your POM.

Maven by default uses : 

```
    <resources>
      <resource>
        <directory>${project.basedir}/src/main/resources</directory>
      </resource>
    </resources>
```

https://maven.apache.org/ref/3.9.5/maven-model-builder/super-pom.html

In Spring Boot, they modified it to filter application yml/properties files

```
<resources>
<resource>
<directory>${basedir}/src/main/resources</directory>
<filtering>true</filtering>
<includes>
<include>**/application*.yml</include>
<include>**/application*.yaml</include>
<include>**/application*.properties</include>
</includes>
</resource>
<resource>
<directory>${basedir}/src/main/resources</directory>
<excludes>
<exclude>**/application*.yml</exclude>
<exclude>**/application*.yaml</exclude>
<exclude>**/application*.properties</exclude>
</excludes>
</resource>
</resources>
```

https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-starter-parent/3.1.5/spring-boot-starter-parent-3.1.5.pom

And AFAIU in JHipster you were filtering YML files under the config folder

I suppose you want to continue to filter these files to keep the compatibility

Reply:
Can you please make a suggested change so I know what you mean?

Reply:
@mraible I did another PR, it was easier to do it from my IDE: #23946

I found something which should remove the warning while keeping exactly the same behavior than today. 

(I sadly cannot test it myself easily right now)